src/python/scripts/dx-docker

```diff
@@ -210,9 +210,11 @@ def run(args):
 
     container_cmd = entrypoint + cmd
 
-    workdir = args.workdir
+    workdir = "/"
     if 'workingDirectory' in imagemeta['app']:
        workdir = imagemeta['app']['workingDirectory']
+    if args.workdir:
```

Comment:
For this container, the working directory (or at least the one containing `env_wrapper.sh`) is the home folder of our non-root container user, `~viral-ngs-user`.